.github/workflows/intl-data-tests.yml

```diff
@@ -0,0 +1,63 @@
+name: Intl data tests
+
+on:
+    push:
+    pull_request:
+
+jobs:
+
+    intl-data-check:
+        name: Check if required
+        runs-on: ubuntu-latest
+        outputs:
+          modified: ${{ steps.intl-data-check.outputs.modified}}
+
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+                with:
+                    fetch-depth: 2
+
+            -   name: Check if intl data files were modified
+                id: intl-data-check
+                run: |
+                    (git diff --name-only HEAD^ HEAD | grep src/Symfony/Component/Intl/Resources/data) &&
```

Comment:
Couldn't this be done on top 🔝 using:

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths

?

Reply:
Nope. That would make that tests are always executed as these paths always exist.

Reply:
However, reading the following paragraph, it should work. I've tested it and it didn't... let me try again

<img width="782" alt="image" src="https://user-images.githubusercontent.com/190447/101400087-4ceac100-38c8-11eb-83da-92fc91af4327.png">

Reply:
Yeah, that worked this time... not sure what I did wrong the first round. Thanks for making me try it again!

Reply:
Great!